src/wkwebkit.cs

```diff
@@ -297,6 +297,14 @@ interface WKWebsiteDataStore : NSCoding {
 		[Async]
 		void RemoveDataOfTypes (NSSet<NSString> websiteDataTypes, NSDate date, Action completionHandler);
 	}
+
+	[Mac (10,12)]
```

Comment:
The whole type is marked with:

[Mac (10,10, onlyOn64 : true)]

but no harm in double marking the 64-bit only ness.